main/gdisk/patches/makefile.patch

```diff
@@ -0,0 +1,10 @@
+--- Makefile	2023-10-21 00:13:47.131679080 +0300
++++ Makefile	2023-10-21 00:46:48.279265063 +0300
+@@ -162,6 +162,8 @@
+ lint:	#no pre-reqs
+ 	lint $(SRCS)
+ 
++check: 	#empty 
++
```

Comment:
if there's no check just do `options = ["!check"]` and you don't need this

Reply:
also it seems like the test is in `builddir/gdisk_test.sh`, so you can run that from `self.do` when overriding `do_check`.

Reply:
ah, there is a target for it. you can do `make_check_target = "test"`

Reply:
(but that runs it with bash, when you can manually call `sh` on it and it probably works, if you want to skip the dep)

Reply:
> if there's no check just do `options = ["!check"]` and you don't need this

ERROR: lint failed: check disabled but no reason given